src/components/views/signin/components/TrezorBridge/TrezorBridge.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { compose, setDisplayName, pure, withState, withHandlers } from 'recompose'
+import { css } from 'aphrodite/no-important'
+import Select from '../../../../elements/Select'
+import tx from '../../../../../theme/styles'
+import sx from './styles'
+import { stateSpec, handlers } from './lib'
+import { DownloadButton } from './components'
+
+const displayName = 'components/views/signin/components/TrezorBridge'
+
+const enhance = compose(
+  setDisplayName(displayName),
+  ...stateSpec.map(statePropDefinition => withState(...statePropDefinition)),
+  withHandlers(handlers),
+  pure
+)
+
+function TrezorBridge (props) {
+  const { hidden, selectedOS, handleSelectOS } = props
```

Comment:
abbreviations should have only first letter capitalized, consider renaming to `selctedOs` and keeping this convention